platform/darwin/src/MGLLocationManager.h

```diff
@@ -0,0 +1,103 @@
+#import <Foundation/Foundation.h>
+#import <CoreLocation/CoreLocation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol MGLLocationManagerDelegate;
+
+/**
+ The `MGLLocationManager` protocol defines a set of methods that you
+ use to receive location-related events.
+ */
+@protocol MGLLocationManager <NSObject>
+
+@required
+
+/**
+ The delegate to receive updates.
```

Comment:
Note that the developer _must not_ set this property before or after making this object the map view’s location manager.